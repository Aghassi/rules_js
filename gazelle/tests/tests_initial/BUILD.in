load("@npm//@bazel/typescript:index.bzl", "ts_project")

ts_project(
    name = "tests_initial",
    srcs = ["main.ts"],
)