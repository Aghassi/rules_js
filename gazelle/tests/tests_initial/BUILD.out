load("@npm//@bazel/typescript:index.bzl", "ts_project")

ts_project(
    name = "tests_initial",
    srcs = ["main.ts"],
)

ts_project(
    name = "tests_initial_tests",
    srcs = ["main.spec.ts"],
    deps = [":tests_initial"],
)
